plugins/content/caption/base/lib/kCaptionsContentManager.php

```diff
@@ -207,6 +207,18 @@ public static function parseStrTTTime($timeStr)
 		return array ($timeInMilliseconds, $error);
 	}
 
+	/**
+	 * @param $time
+	 * @return string
+	 */
+	public function parseCaptionTime($time)
+	{
+		if($this instanceof srtCaptionsContentManager)
+			$time = str_replace(',','.',$time);
```

Comment:
Either fix the time string before calling this function , or override this entire function in srt